docker/client/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+yarn install
```

Comment:
if you do this, you dont have any guarantee that the local version applied is the same versions that will be applied in the docker container... there's a reason why we use `--frozen-lockfile`, they are more stable